docs/index.rst

```diff
@@ -3,14 +3,15 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to SecureDrop Workstation's documentation!
-==================================================
+Welcome to the SecureDrop Workstation documentation!
+====================================================
 
 .. include:: includes/top-warning.rst
 
-SecureDrop Workstation is an application built on Qubes OS, that can be used in
-conjunction with SecureDrop to communicate with anonymous sources and manage
-submitted documents with a reasonable level of security.
+SecureDrop Workstation is an application built on Qubes OS to enable journalists
```

Comment:
Maybe "system," "solution," or "environment"? "Application" seems limited in this context.

Reply:
I question if those points will make sense to anyone but SD admins. The highly technical language feels inappropriate for the index page. Also, as I've been aware, the "Workstation" is the machine—not the client application. The "SecureDrop app" needs to be spoken to in that terminology, if this is to make sense to journalist/non-technical users.

Reply:
I've simplified as follows:

> SecureDrop Workstation is a tool to enable journalists to communicate with anonymous sources and manage submitted documents, while providing mitigations against malware and other security risks. It is built on Qubes OS and requires a SecureDrop server setup.

Reply:
Is it an option for technical,multi-syllabic words such as "mitigations" and "provision" be kept out of the text intended for Journalists to read? I'm aware they're all very intelligent ppl; however mitigating against impostor syndrome and instilling confidence in users feels as important as providing information to them.

Reply:
What would you suggest in this instance? One thing I want to be a bit wary of is over-promising -- "malware protection" for example has specific meaning to folks, and typically includes stuff we don't do (e.g., scanning files for viruses).

Reply:
Not sure; can I mull over the weekend (I need to refresh my pull on my machine, and leave for the day in an hour)? I'm also concerned that there's inconsistency in referring to the SecureDrop Workstation as the app vs the machine.

Reply:
Suggestions welcome, but as we discussed just now, I also think it makes sense to talk through the challenges of not triggering impostor syndrome, and of writing docs accessible to journalists, in a higher level review sometime next week.  To be clear, I agree these are very important issues in writing good, _effective_ documentation for this project. :)